slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -146,9 +147,13 @@ public void exportDataWithFilter(final OutputStream output, final List<String> p
 
   @Override
   public IncrementalExporter createIncrementalExporter(final OutputStream out) {
+    if (!gvrValidator.genesisValidatorRootExists()) {
+      return new NoOpIncrementalExporter(out);
+    }
```

Comment:
nit: might be nice to create the NoOpIncrementalExporter at the same level as abstraction as before, e.g. by exposing genesisValidatorRootExists to the level above.

The reader would also benefit from the existing comment that explains why we use the noop exporter:
>  // Using no-op exporter instead of returning an optional so can use try with for closing

Reply:
Added a comment. However, exposing checkGVR at SlashingProtection interface would add two level of abstraction i.e. DBSlashing.checkGVR -> gvrValidator.checkGVR. T